tutorial/image-focus-point.md

```diff
@@ -0,0 +1,98 @@
+---
+title:  Implementing Focus Point using Gumlet 
+parent: Malibu Tutorial
+nav_order: 27
+nav_exclude: true
+---
+
+# {{page.title}}
+
+_This tutorial was contributed by [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
+
+You can set a focal point to ensure that a key part of an image won't get cropped. Focus points are particularly useful for responsive layouts and make sure the most important part of the image is always visible regardless of the element size. It will ensure that embarrassing image crops do not happen.
+
+## How to set image Focus Point in Bold
+
+![Image Focus Point]({{"images/set-image-focus-point.gif" | absolute_url}})
+
+After you upload an image, you can click anywhere on that image to define a Focus Point for that image. Focus Point ensures that when an image gets cropped to different sizes, the set Focus Point of the image will always be in the viewport.
```

Comment:
This "the set Focus Point of the image will always be in the viewport" can be changed to "the set Focus Point will always be in the viewport."

Reply:
done